src/lib/orion_websocket/ws.cpp

```diff
@@ -176,6 +182,21 @@ void *runWS(void *ptr)
     pthread_mutex_lock(&mtx);
     lws_service(ws->ctx, 50);
     pthread_mutex_unlock(&mtx);
+/*
```

Comment:
(Pls, remember the "Fixed in X" message. Otherwise, I don't have any mean to know that this has been solved, apart from the costly process or reviewing the whole PR again -easy in this case, as the PR is short, but not in a general case, as the PR could include hundred of lines changed. Thanks!)